src/DETHRACE/common/car.c

```diff
@@ -5639,8 +6828,55 @@ int DoPullActorFromWorld(br_actor* pActor) {
     tNon_car_spec* non_car;
     LOG_TRACE("(%p)", pActor);
 
-    STUB();
-    return 0;
+    non_car = NULL;
+    num = pActor->identifier[2] + 2 * (5 * pActor->identifier[1] - 240) - 48;
```

Comment:
```suggestion
    num = 10 * (pActor->identifier[1] - '0') + 1 * (pActor->identifier[2] - '0');
```

Reply:
how did you make that leap? I had to do the math just now to prove to myself they are equivalent ;)

Reply:
Trial and error, I guess.
By looking over and over again at an ascii table.
By know, I know by heart 0x30 is start of decimal numbers, 0x40 for uppercase letters and 0x60 for lowercase letters.